File: FoodDelivery_Backend/Models/SupplierInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FoodDelivery_Backend.Models
{
    public class SupplierInfo
    {
        public decimal supplier_id { get; set; }
        public string supplier_name { get; set; }
        public string supplier_address { get; set; }
        public decimal supplier_gst_num { get; set; }
        public string reg_date { get; set; }
        public decimal pincode { get; set; }
        public SupplierType ST { get; set; }
        public string supplier_status { get; set; }
        public string longtitude { get; set; }
        public string latitude { get; set; }
        public string serv_pin_list { get; set; }


    }
}

File: FoodDelivery_Angular_Frontend/Frontend-App/src/app/Models/supplierInfo.ts
import { supplierTypeModel } from "./supplierType";

export class supplierInfoModel {
    supplier_name: string;
    latitude: string;
    longtitude: string;
    pincode: number|null;
    reg_date: string;
    serv_pin_list: string;
    supplier_address: string;
    supplier_gst_num: number|null;
    supplier_type: string;
    supplier_id: number;
    isEdit: boolean;
    supplier_status:string;
    ST:supplierTypeModel;



    constructor() {
        this.supplier_name = '';
        this.latitude = '';
        this.longtitude='';
        this.pincode = null;
        this.reg_date = '';
        this.serv_pin_list = '';
        this.supplier_address = '';
        this.supplier_status = '';
        this.supplier_gst_num = null;
        this.supplier_type = '';
        this.supplier_id = 0;
        this.isEdit=false;
        this.ST=new supplierTypeModel();

    }
}
